applications/vanilla/models/class.commentmodel.php

```diff
@@ -861,6 +861,18 @@ public function setWatch($discussion, $limit, $offset, $totalComments) {
                     ]
                 );
             }
+        } else {
+            // Insert watch data.
+            $this->SQL->options('Ignore', true);
+            $this->SQL->insert(
+                'UserDiscussion',
+                [
+                    'UserID' => $userID,
+                    'DiscussionID' => $discussion->DiscussionID,
+                    'CountComments' => $countWatch,
+                    'DateLastViewed' => Gdn_Format::toDateTime()
```

Comment:
Any particular reason we opted for using this deprecated method? Suggested alternative now is `DateTimeFormatter::timeStampToDateTime()`.

Reply:
> Any particular reason we opted for using this deprecated method?

This was restoring erroneously-deleted code, verbatim. However, since I have technically touched the code, I have updated the deprecated method usage.